htdocs/core/lib/functions.lib.php

```diff
@@ -1154,7 +1154,7 @@ function dol_buildpath($path, $type = 0, $returnemptyifnotfound = 0)
  *	@return object				Clone object
  *  @see https://php.net/manual/language.oop5.cloning.php
  */
-function dol_clone($object, $native = 0)
+function dol_clone($object, $native = 2)
```

Comment:
Changing a default value can be done in develop branch only.

Reply:
OK thank you, I'll switch branch :pray: